### PR TITLE
worked on que page

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
image
Kindly review the quiz que page. Let me know what do you think about it.

### CODE REVIEW

File: Question/Question.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	background-color: var(--lightred-color);
+	color:var(--darkred-color);
+	border-color: var(--darkred-color);
+}
+
+.quiz-home-box .option-box .option.wrong::before{
+    content: '';
+    position: absolute;
+    background-color: transparent;
+    height: 3rem;
```
Comment (Sahil25061999) [inline Question/Question.css:162]: since most of the code is repeating here you can just use give this properties together

**Review by Sahil25061999 — APPROVED** [review]
remove commented code if not necessary, also since the font family is being repeated you can just create a separate class for it.

File: Question/Question.css [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+	padding: 1rem;
+	border-radius: 10px;
+    text-decoration: solid;
+    overflow-wrap: break-word;
+}
+
+.quiz-home-box .next-question{
+    display: flex;
+    /* justify-content: center;
+    align-items: center; */
```
Comment (vanshitamoolchandani) [inline Question/Question.css]: Remove comments and extra space

File: Question/Question.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                <div class="option">option 3</div>
+                <div class="option wrong">option 4</div>
+            </div>
+            <div class=" col-6 answer-description">
+                They really don'tvdfnvkdfnjkvnfdjknvjkfdnjkvnjfdknvjkdfnsjkvnjkfdnjkvnjkdsfnjkvnsjkdfnvjknsdfjkvjksdnfjkvsjkdfnjkvnjksdfnvjkndfj vsdhgvcghsdvghcvsdghvcghvagsdcghvdsghjcvghdsvghcvghadscghvdsghcvghdsvghcvghdsvcghavghsdhgv
+            </div>
+    
+            <div class="col-6 next-question">
+                <button type="button" class="btn btn-sm btn-primary btn-next-que">Next Question</button>
+                <a href="../Categories/Categories.html" class="btn text-decoration-none btn-sm btn-link btn-see-result">See Your Result !</a>
```
Comment (vanshitamoolchandani) [inline Question/Question.html:64]: Small UI suggestion, usually 'next question' button is on the right side of the page. Other than this you are good to merge

**Review by vanshitamoolchandani — APPROVED** [review]

File: Question/Question.css (hunk 1 above)
Comment (kgajriaAgarwal) [reply Question/Question.css:162]: only 2 -3 properties are repeating , rest all are different..

**Pull request merged**